frontend/src/app/helpers/mat-icon/mat-icon.component.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-mat-icon',
+  templateUrl: './mat-icon.component.html',
+  styleUrls: ['./mat-icon.component.css'],
+})
+export class MatIconComponent implements OnInit {
+  @Input() position: MatIconPosition = null;
+  @Input() size: string = '24px';
+
+  @Input() usage: MatIconUsage = 'general';
+  style = {};
+
+  constructor() {}
+
+  ngOnInit(): void {
+    this.style = {
+      width: this.size,
+      height: this.size,
+      'font-size': this.size,
+      position: 'relative',
+      top: this.mapUsageToCSSRelativeTop(this.usage),
+      'margin-right': '0px',
+    };
+  }
+
+  mapUsageToCSSRelativeTop(usage: MatIconUsage): string {
```

Comment:
The size of the icon should be relative to the police size for button and text, so use units such as `em` rather than `px`. See https://www.w3.org/Style/Examples/007/units.en.html